<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="pc-PC" ignored="0" name="test1" tests="28" failures="4" timestamp="10 七月 2019 13:38:17 GMT" time="82.918" errors="0">
  <testcase name="f" time="5.96" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="2.263" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="0.318" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="0.282" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="0.14" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="1.129" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="13.387" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="0.106" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="0.949" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="0.372" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="0.181" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="1.002" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="0.199" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="1.0" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="0.145" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="0.254" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="0.186" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="0.296" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="0.531" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="0.635" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="3.979" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="3.976" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="0.75" classname="com.testing.testng.dataDriven"/>
  <testcase name="f" time="11.279" classname="com.testing.testng.dataDriven">
    <failure type="java.lang.AssertionError" message="expected [pass] but found [fail]">
      <![CDATA[java.lang.AssertionError: expected [pass] but found [fail]
at com.testing.testng.dataDriven.f(dataDriven.java:42)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- f -->
  <testcase name="f" time="11.12" classname="com.testing.testng.dataDriven">
    <failure type="java.lang.AssertionError" message="expected [pass] but found [fail]">
      <![CDATA[java.lang.AssertionError: expected [pass] but found [fail]
at com.testing.testng.dataDriven.f(dataDriven.java:42)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- f -->
  <testcase name="f" time="10.605" classname="com.testing.testng.dataDriven">
    <failure type="java.lang.AssertionError" message="expected [pass] but found [fail]">
      <![CDATA[java.lang.AssertionError: expected [pass] but found [fail]
at com.testing.testng.dataDriven.f(dataDriven.java:42)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- f -->
  <testcase name="f" time="11.089" classname="com.testing.testng.dataDriven">
    <failure type="java.lang.AssertionError" message="expected [pass] but found [fail]">
      <![CDATA[java.lang.AssertionError: expected [pass] but found [fail]
at com.testing.testng.dataDriven.f(dataDriven.java:42)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- f -->
  <testcase name="f" time="0.618" classname="com.testing.testng.dataDriven"/>
</testsuite> <!-- test1 -->
